query/expression.go

```diff
@@ -46,34 +116,34 @@ type EvaluationContext struct {
 // Expressions correspond to the timerange in the current EvaluationContext.
 type Expression interface {
 	// Evaluate the given expression.
-	Evaluate(context EvaluationContext) (*api.SeriesList, error)
+	Evaluate(context EvaluationContext) (Value, error)
+}
+
+func EvaluateToSeriesList(e Expression, context EvaluationContext) (api.SeriesList, error) {
```

Comment:
Although I haven't done it yet, I think I'm going to use it to wire up aggregation/transformation functions, so I'm going to leave it where it is for now